Services/Data/default.json

```diff
@@ -93,27 +95,29 @@
       ]
     },
     {
-      "Id": "default_Engine_Fuel_Empty",
       "Name": "Engine tank empty",
       "Enabled": true,
       "Description": "Fuel level is less than 5",
-      "GroupId": "default_Engines",
-      "Severity": "info",
+      "GroupId": "Engines",
+      "Severity": "Info",
+      "Calculation": "Instant",
+      "TimePeriod": "00:00:00",
       "Conditions": [
         {
-          "Field": "fuellevel",
+          "Field": "fuel",
```

Comment:
This should be actually `fuellevel`

Reply:
should we change this for telemetry service too?